packages/abi-gen-wrappers/src/generated-wrappers/asset_proxy_owner.ts

```diff
@@ -234,20 +210,19 @@ export class AssetProxyOwnerContract extends BaseContract {
         awaitTransactionSuccessAsync(
             owner: string,
             txData?: Partial<TxData>,
-            pollingIntervalMs?: number,
-            timeoutMs?: number,
+            opts: TxOpts = { shouldValidate: true },
         ): PromiseWithTransactionHash<TransactionReceiptWithDecodedLogs> {
             assert.isString('owner', owner);
             const self = (this as any) as AssetProxyOwnerContract;
-            const txHashPromise = self.addOwner.sendTransactionAsync(owner.toLowerCase(), txData);
+            const txHashPromise = self.addOwner.sendTransactionAsync(owner.toLowerCase(), txData, opts);
             return new PromiseWithTransactionHash<TransactionReceiptWithDecodedLogs>(
                 txHashPromise,
                 (async (): Promise<TransactionReceiptWithDecodedLogs> => {
                     // When the transaction hash resolves, wait for it to be mined.
                     return self._web3Wrapper.awaitTransactionSuccessAsync(
                         await txHashPromise,
-                        pollingIntervalMs,
-                        timeoutMs,
+                        opts.pollingIntervalMs,
+                        opts.timeoutMs,
```

Comment:
I intentionally didn't change `web3wrapper` because the arguments for `awaitTransactionSuccessAsync(txHash, pollingIntervalMs, timeoutMs)` don't have overlap with `sendTransactionAsync(txData)`. They are pretty different steps.

`awaitTransactionSuccessAsync` in the *contract wrappers* has been updated to take `TxOpts` though.

In the contract-wrapper class, `awaitTransactionSuccessAsync` is a convenience that calls `web3Wrapper.sendTransactionAsync(txData)`, retrieves the txhash, then calls `web3Wrapper.awaitTransactionSuccessAsync(txHash)`. Thus `sendTransactionAsync()` is a subset of `awaitTransactionSuccessAsync()`. I created the `TxOpts` object to 1. further abstract the differences between the two away so a developer can call both with the same arguments, and 2. allow us more flexibility to add or remove optional args later.